Add Enphase Envoy multi-phase sensor description (#30818)
home-assistant/home-assistant.github.io: source/_integrations/enphase_envoy.markdown

- Historical energy production & consumption
- Power production per-inverter

> Production and consumption sensors for each phase are available for Envoy-metered with installed and configured current transformers (CT) on more than 1 phase.

Reviewer: ```suggestion
Production and consumption sensors for each phase are available for Envoy S Metered with installed and configured current transformers (CT) on more than 1 phase.
```
On their website, I found Envoy S Metered and Envoy S Metered Kit, without a dash. Is this the product you're referring to here?

Author: Yes. And since they renamed it to IQ Gateway, [IQ Gateway Metered](https://enphase.com/en-eu/store/communication/iq-gateway-metered) as well. You find still both names in use.

Reviewer: Oh, I see. In that case, would it be worth adding both names?

Author: We could do Envoy S Metered / IQ Gateway metered.

Reviewer: sounds good!

> Production and consumption sensors for each phase are available for Envoy S Metered with installed and configured current transformers (CT) on more than 1 phase.

Reviewer: ```suggestion
Production and consumption sensors for each phase are available for Envoy S Metered / IQ Gateway Metered with installed and configured current transformers (CT) on more than 1 phase.
```

> For Envoy-metered with installed and configured current transforemrs (CT) on more then 1 phase, production and consumption sensors for each individual phase are available.

Reviewer 2: ```suggestion
Production and consumption sensors for each phase are available for Envoy-metered with installed and configured current transformers (CT) on more than 1 phase.
```
